application/helpers/export_helper.php

```diff
@@ -1399,13 +1399,37 @@ function quexml_set_default_value(&$element, $iResponseID, $qid, $iSurveyID, $fi
         if ($colname != "") {
             // prepare and decrypt data
             $oResponse = Response::model($iSurveyID)->findByPk($iResponseID);
-            $oResponse->decrypt(); 
+            $oResponse->decrypt();
             $value = $oResponse->$colname;
             $element->setAttribute("defaultValue", $value);
         }
     }
 }
 
+/**
+ * Format defaultValue of Date/Time questions according to question date format
+ *
+ * @param mixed $element DOM element with the date to change
+ * @param int $qid The qid of the question
+ * @param int $iSurveyID The survey id
+ */
+function quexml_reformat_date(&$element, $qid, $iSurveyID)
```

Comment:
If you can add a proper type-hint rather than `mixed`, that would be good, too.

Reply:
Sure, I will...Sorry about that

Reply:
> Objects are always sent by reference, you don't need `&`.
> If you can add a proper type-hint rather than mixed, that would be good, too.
Both are consistent with other function definitions, like quexml_set_default_value

I know it is not the best, but shouldn't we keep it consistent?

Reply:
We should keep it correct. ;) Consistency might suffer a bit as we improve the code bit by bit.